File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.py
n = int(input())

for i in range(n):
    sum = 0
    point = 0
    ox = input()
    for i in ox:
        if i == 'O':
            point+=1
            sum+=point
        else:
            point=0
    print(sum)
        

    
    

File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.java
import java.util.Scanner;
class Main{
	public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        
        for(int i = 0; i<n ; i++){
            String ox = sc.next();
            int point = 0;
            int sum = 0;
            for(int j=0;j<ox.length();j++){
                if(ox.charAt(j)=='O'){
                    point+=1;
                    sum+=point;
                }else point=0;
            }
            System.out.println(sum);
        }
		
	}
}
